src/ImageBackground.tsx

```diff
@@ -0,0 +1,12 @@
+import { ImageProps } from './Image';
```

Comment:
[ImageBackground](https://facebook.github.io/react-native/docs/imagebackground) is a react-native's official component.

So, do the following instead:

`import { ImageBackgroundProps } from 'react-native';`